### ISSUE TITLE
최신 코드 혹은 특정 버전 사용 방법

### ISSUE THREAD

**Issue by changdoc** [changdoc/naver-vaccine-macro#29]
크롭 웹스토어는 심사 과정이 있어 버전 패치가 즉시 이루어지지 않고 있어 github에 있는 소스코드를 사용하는 방법을 공유드립니다.

최신 코드 사용 방법
<img width="1186" alt="스크린샷 2021-07-19 오후 12 40 02" src="https://user-images.githubusercontent.com/1879494/126099761-b94c6b31-d1b2-4653-a7b0-559a11ae51bd.png">

특정 버전 사용 방법
<img width="585" alt="스크린샷 2021-07-19 오후 12 40 31" src="https://user-images.githubusercontent.com/1879494/126099799-33a54ea9-9e4b-4cb9-8463-80417ac8cf51.png">
<img width="675" alt="스크린샷 2021-07-19 오후 12 40 22" src="https://user-images.githubusercontent.com/1879494/126099804-5b23278d-1b0a-402c-a30c-6e143d165ea7.png">

다운 받은 최신 코드, 혹은 특정 버전 zip 파일 위치를 기억해두고 아래 블로그 설명 대로 따라하시면 됩니다.
(크롬 개발자 모드 관련 설명 블로그) https://trend21c.tistory.com/1030

**Comment by koolingo** [comment]
알려주신 방법대로
개발자 모드 → 압축해제된 확장 프로그램 로드 클릭 → 다운받은 소스코드 불러오기(확장프로그램 로드됨 메세지) →  업데이트 버튼 클릭 (업데이트중, 확장 프로그램 업데이트됨 메세지)

위와 같이 적용한후  병원목록 추출했으나 동일증상으로 잘못된 접근이라고 나옵니다.
혹시 제가 놓친 부분이나 실수한 부분이 있을까요?

**Comment by changdoc** [comment]
아하 크롬 스토어에서 설치한 버전은 삭제해야 되겠네요

**Comment by gazuahwasung** [comment]
안녕하세요 개발자님~! 잔여백신 예약하기가 넘 어려워서 깃헙이란 사이트를 예전에 블록체인에 관심이 많아서 개발자들의 사이트라고 알고있어서 서칭했더니 나와서 차근차근 스텝스텝 밟아서 따라와서 결구 자동예약시작 버튼까지 활성화 시켰습니다. 여기서 궁금증이 몇가지 생겼는데 1. 텔레그램 봇을 연동시킨 이유가 뭔지도 궁금하고(물론 텔레그램 bot id와 chat id가 뭔지 몰라서 찾아서 셋팅했습니다.  또 자동예약시작 버튼을 누르면 어떤 구조로 예약이 되는지 궁금합니다. 그 특정 병원 예약 사이트에서 2초마다 새로고침되서 백신이 뜨게되면 백신종류 선택 + 백신 예약완료버튼이 눌러지는 구조인가요? 그러면 여기서 텔래그램 에약 알림을 연동한 이유도 궁금합니다. 그냥 잔여백신이 나왔다고 알려주는 역할을 하는 정도인가요?

**Comment by changdoc** [comment]
네 동작 방식은 말씀하신것 처럼 예약 신청 페이지에서 주기적으로 새로고침을 수행 하다 잔여 백신 수량이 존재한다고 판단될 경우 예약신청을 자동으로 진행 합니다

텔레그램 연동은 예약 시도를 하는 상황에서 화면을 보고 있지 않을때도 알림을 받기 위한 방법으로 보시면 될것 같네요
시도 후 결과나 실패나 에러가 나는 상황을 구분지어 메시지가 발송되게 됩니다

메시지 발송이 필요 없다면 초기화를 눌러 연동한 정보를 제거 하면 메시지 발송은 하지 않습니다

**Comment by gitlsc013** [comment]
개발자님 자동예약 시작버튼이 안보여요

**Comment by gazuahwasung** [comment]
@gitlsc013  개발자는 아니지만 자동예약 시작 버튼이 그 병원 목록 추출하셔서 특정 병원 선택하셔서 링크까지 열었는데도 안보이시는건가요? 올려주신 설명대로 차근차근 따라가니깐 전 지금 자동예약 시작 버튼 보여서 누르고 대기중인데ㅠ

**Comment by doreme1** [comment]
ㅠㅠ 저도 거기까지 했는데 자동예약버튼이 안보이네요

**Comment by gazuahwasung** [comment]
@doreme1 예약페이지안에 우상단 1시방향에 자동예약버튼있는데ㅠ 프로그램은 저는 구글 크롬에서 확장프로그램앱으로 열었어요.

**Comment by gitlsc013** [comment]
아 떳어요 확장프로그램자체에 있는게아니라 예약사이트 화면에 자동예약버튼이 있는거구나 감사합니다

**Comment by baruegen** [comment]
개발자님 1.0.9 버전 쓰는데 병원 링크 열기하면 '잠시 후 다시 시도하세요.' 문구가 뜨는데 어떤걸 놓친건가요? 이전까지 했을 때는 문제없이 예약을 했는데 궁금합니다. (웨일, 크롬) 둘 다 해봤습니다!

**Comment by OneTop4458** [comment]
> 개발자님 1.0.9 버전 쓰는데 병원 링크 열기하면 '잠시 후 다시 시도하세요.' 문구가 뜨는데 어떤걸 놓친건가요? 이전까지 했을 때는 문제없이 예약을 했는데 궁금합니다. (웨일, 크롬) 둘 다 해봤습니다!

질문 답변은 이제 다음 링크에서 관리 됩니다.
https://github.com/changdoc/naver-vaccine-macro/discussions/38
